docs/account/my-organization/role-management.md

```diff
@@ -19,210 +19,333 @@ tags:
   ]
 ---
 
-### Build Profile Permissions
+# Role Management
 
-Permissions can be customized for build profiles.
+With Appcircle's [**Advanced Role Management**](/account/my-organization#advanced-role-management) structure, you can detail the roles of the members you add to your organisation separately for each module and restrict the authority of the members. Appcircle offers different role types for each module. A short description of each role is given in the table below. In addition, you can find detailed role management information for each module under module titles.
 
-| Permission | Explanation                                                                                            |
-| ---------- | ------------------------------------------------------------------------------------------------------ |
-| Manager    | The user can view and download logs, change configuration, workflows, and triggers and start building. |
-| Operator   | The user can only start the build, view logs, and download logs.                                       |
-| Viewer     | The user can only view and download logs.                                                              |
-| None       | The user cannot reach any details about build profiles.                                                |
+- **Owner**: The user is authorised unlimited access to all modules.
+- **Manager**: The user becomes the administrator of the relevant module and no restrictions are made.
+- **Operator**: The user is the person who manages the operations of the relevant module and certain restrictions apply.
+- **Ext. Operator**: The user has very limited authorisation in the relevant module. It is usually for third party employees from outside the company.
+- **Viewer**: User only has view authorisation in the relevant module, cannot take any action.
 
-:::info
-
-Manager or Operator Build Profile permission can distribute binary if user has Manager or Operator Distribution permission.
-
-:::
-
-:::info
+:::caution Role Types
 
-Manager or Operator Build Profile permission can publish if user has Manager or Operator Publish Android/iOS permission.
+Some role types are not used in some modules because they are not required in that module because they are the same as another role. Therefore, roles may differ for each module.
```

Comment:
```suggestion
Some role types are not used in certain modules because they are redundant or unnecessary, as they serve the same function as another role. Therefore, roles may vary for each module.
```